src/frontends/paddle/src/internal/pass/transform_fakequantize.cpp

```diff
@@ -0,0 +1,132 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "internal/pass/transform_fakequantize.hpp"
+
+#include <ngraph/ngraph.hpp>
+#include <ngraph/op/util/op_types.hpp>
+#include <ngraph/pattern/matcher.hpp>
+#include <ngraph/pattern/op/or.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/rt_info.hpp>
+
+#include "default_opset.hpp"
+#include "openvino/pass/pattern/op/label.hpp"
+#include "transformations/utils/utils.hpp"
+
+using namespace ov::frontend::paddle::op::default_opset;
+using namespace ov;
+using namespace ov::pass;
+using namespace ov::frontend::paddle::op;
+
+/*
+                                  zero_point
+                                      /
+                        input    convert     scale
+                           \        /         /
+                            subtract  Multiply
+   quantize_linear   ==>>        \      /
+                                  Divide
+                                     \
+                                      Round
+                                         \
+                                          Clamp
```

Comment:
If you ignore the rounding mode, could you confirm that this part can be expressed as FQ node with some well-defined subexpressions taken on inputs?